doc/user-guides/ratelimiting/authenticated-rl-for-app.md

```diff
@@ -1,4 +1,4 @@
-# Authenticated Rate Limiting for Application Developers
+# Authenticated Rate Limiting for Applications
```

Comment:
I think having the persona might put people off but happy to change

Reply:
It is referring to the persona defined by Gateway API https://gateway-api.sigs.k8s.io/